### ISSUE TITLE
[BUG] enableAuthorIndependent

### ISSUE THREAD

**Issue by i6h** [acegoal07/discordjs-pagination#17]
**Describe the bug**
enableAuthorIndependent not work when i type the command the button dont work

**Error message**
``` no Error```

**Sample of the code where it is used**
``` js
    await interaction.deferReply()
    new Pagination().setPortal(interaction)
    .setPageList(embeds)
    .setButtonList([
      new ButtonBuilder()
      .setEmoji(`◀`)
      .setStyle("Secondary")
      .setCustomId(`1`),
      new ButtonBuilder()
      .setEmoji(`▶`)
      .setStyle("Secondary")
      .setCustomId(`2`)
    ])
    .enableAuthorIndependent()
    .setTimeout(15000)
    .paginate()
```

**Additional info**
- Package version: ``` 1.5.2 ```
- Enabled settings: ``` enableAuthorIndependent ```

**Comment by acegoal07** [comment]
Was the setting working before the recent update?

**Comment by acegoal07** [comment]
@i6h When did you start having this issue?

**Comment by acegoal07** [comment]
Fixed in version 1.5.3
